src/sample/courseRegController.java

```diff
@@ -50,50 +50,82 @@ public void openBrowser() throws URISyntaxException, IOException {
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
-    registeredCourse.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
+    registeredCourse.getItems().add("CMT221 - DATABASE ORGANISATIONS AND DESIGN");
+    registeredCourse.getItems().add("CST232 - OPERATING SYSTEMS");
+    registeredCourse.getItems().add("CSE241 - FOUNDATION OF SOFTWARE ENGINEERING");
+    registeredCourse.getItems().add("CPT111 - PRINCIPLES OF PROGRAMMING");
+    registeredCourse.getItems().add("CPT112 - DISCRETE STRUCTURES");
+    registeredCourse.getItems().add("CST131 - COMPUTER ORGANISATIONS");
+    registeredCourse.getItems().add("CPT113 - PROGRAMMING METHODOLOGY AND DATA STRUCTURES");
+    registeredCourse.getItems().add("CPT115 - MATHEMATICAL METHODS FOR COMPUTER SCIENCE");
+    registeredCourse.getItems().add("CPC151 - FUNDAMENTALS OF LOGIC AND ARTIFICIAL INTELLIGENCE");
+    registeredCourse.getItems().add("AKW103 - INTRODUCTION TO MANAGEMENT");
+    registeredCourse.getItems().add("AKW104 - ACCOUNTING AND FINANCE");
+    registeredCourse.getItems().add("ATW202 - BUSINESS RESEARCH METHOD");
+    registeredCourse.getItems().add("ATW241 - PRINCIPLES OF MARKETING");
+    registeredCourse.getItems().add("ATW262 - PRINCIPLES OF FINANCE");
+    registeredCourse.getItems().add("MAA101 - CALCULUS FOR SCIENCE STUDENTS");
+    registeredCourse.getItems().add("MAT111 - LINEAR ALGEBRA");
+    registeredCourse.getItems().add("MAT181 - PROGRAMMING FOR SCIENTIFIC APPLICATIONS");
+    registeredCourse.getItems().add("MAA111 - ALGEBRA FOR SCIENCE STUDENTS");
+    registeredCourse.getItems().add("LAK100 - KOREAN LANGUAGE");
+    registeredCourse.getItems().add("LAJ100 - JAPANESE LANGUAGE");
+    registeredCourse.getItems().add("LKM400 - BAHASA MALAYSIA");
+    registeredCourse.getItems().add("LSP300 - ACADEMIC ENGLISH");
+    registeredCourse.getItems().add("HFF225 - PHILOSOPHY AND CURRENT ISSUES");
+    registeredCourse.getItems().add("HFE224 - APPRECIATION OF ETHICS AND CIVILISATIONS");
   }
 
   private void suggestcourse() {
     courseSuggestion.getItems().add("CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP");
```

Comment:
why suggestcourse, you need to add yourself?

Reply:
ya this one i havent worked on it yet, i just worked on addcourse so far, for suggestcourse and removecourse i am still figuring out, because the ways to connect is different, will push again once I solved the issue

Reply:
ok